docs/docs/run_minder_server/intro_to_install.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Introduction to running Minder
+sidebar_position: 5
+---
+
+# Introduction to running Minder
+
+Minder is platform, comprising of a control plane, a CLI, a database and an identity provider.
+
+The control plane runs two endpoints, a gRPC endpoint and a HTTP endpoint.
+
+Minder is controlled and managed via the CLI application `minder`.
```

Comment:
```suggestion
Users interact with the Minder server via the CLI application `minder`.
```

The previous sentence suggested that `minder` was used to start and stop the server.